calabash/Classes/FranklyServer/Routes/LPBackdoorRoute.m

```diff
@@ -29,40 +34,41 @@ - (NSDictionary *) JSONResponseForMethod:(NSString *) method URI:(NSString *) pa
     selectorName = [selectorName stringByAppendingString:@":"];
   }
 
+  id argument = [data objectForKey:@"arg"];
+  if (!argument) {
+    LPLogError(@"Expected data dictionary to contain an 'arg' key");
```

Comment:
If we are going to expand the method signatures, then +1 for not requiring an argument _and_ for passing multiple arguments.

I _think_ however that the client should always provide an arg key (or "arguments") key.  But I am not picky.